packages/sdks/snippets/react/src/routes/blueprints/NavLinks.tsx

```diff
@@ -0,0 +1,39 @@
+import { fetchOneEntry, type BuilderContent } from '@builder.io/sdk-react';
+import { useEffect, useState } from 'react';
+
+export default function NavLinks() {
+  const [links, setLinks] = useState<BuilderContent | null>(null);
+
+  useEffect(() => {
+    fetchOneEntry({
+      model: 'navigation-links',
+      apiKey: 'ee9f13b4981e489a9a1209887695ef2b',
+    }).then((data) => {
+      setLinks(data);
+    });
+  }, []);
+
+  return (
+    <nav>
+      {/* Brand name */}
+      <h1>Acme company</h1>
+      {/* Navigation links from Builder */}
+      {links && links.data && (
+        <ul>
+          {links.data.links.map(
+            (link: { url: string; text: string }, index: number) => (
+              <li key={index}>
+                <a href={link.url}>{link.text}</a>
+              </li>
+            )
+          )}
+        </ul>
+      )}
```

Comment:
can you convert this to a component similar to how you've done for all other frameworks in #4019 to maintain consistency